modules/tinymce/src/models/dom/main/ts/table/core/TableWire.ts

```diff
@@ -36,8 +36,7 @@ const getInlineResizeWire = (editor: Editor, isResizable: (elm: SugarElement<Ele
     return ResizeWire.scrollable(editorBody, container, body, isResizable);
   }
 
-  Insert.append(body, container);
-  return ResizeWire.body(editorBody, container, isResizable);
+  return ResizeWire.body(editorBody, isResizable);
```

Comment:
Shouldn't this apply to the split ui scrollable as well? Do we even need a split ui for inline mode if it's defaulting to inside the content?